keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -99,10 +133,16 @@ def pretokenize(text, lowercase, strip_accents, split):
         # Remove the accent marks.
         text = tf.strings.regex_replace(text, r"\p{Mn}", "")
     if split:
+        if split_on_cjk:
+            split_pattern = WHITESPACE_PUNCTUATION_AND_CJK_REGEX
```

Comment:
delim_regex_pattern I guess? since you are agreeing with the other arg name

Reply:
Hehe, I just changed it to `keep_split_pattern` :P

Reply:
that works too!